.travis.yml

```diff
@@ -1,7 +1,13 @@
+
+---
 language: python
-python: 3.6
+python: 2.7
 services: docker
+install:
+- git clone --recursive https://github.com/systemslab/popper /tmp/popper
+- export PATH=$PATH:/tmp/popper/cli/bin
+- export PYTHONUNBUFFERED=1
 script:
-- PATH=$PATH:$PWD/cli/bin docs/ci/demo
-- docker build -t falsifiable/popper cli/
-- docker run --rm -e CI=1 -e POPPER_FIGSHARE_API_TOKEN="${POPPER_FIGSHARE_API_TOKEN}" -e POPPER_ZENODO_API_TOKEN="${POPPER_ZENODO_API_TOKEN}" -e POPPER_GITHUB_API_TOKEN="${POPPER_GITHUB_API_TOKEN}" -v `pwd`/docs/ci/demo:/demo --entrypoint=/demo falsifiable/popper
+  - USE_VIRTUALENV=1 popper run 
```

Comment:
we also need to execute without creating a virtualenv, so that we can test [manual installs](https://github.com/systemslab/popper/tree/master/cli#manual) of popper.

Reply:
Done. Thanks!